programs/witness_node/main.cpp

```diff
@@ -90,10 +91,33 @@ int main(int argc, char** argv) {
       }
       catch (const boost::program_options::error& e)
       {
-        std::cerr << "Error parsing command line: " << e.what() << "\n";
-        return 1;
+         std::cerr << "Error parsing command line: " << e.what() << "\n";
+         return 1;
       }
 
+      if (options.count("plugins")) {
+
+         std::set<std::string> plugins;
+         boost::split(plugins, options.at("plugins").as<std::string>(), [](char c){return c == ' ';});
+
+         if(plugins.count("account_history") && plugins.count("elasticsearch")) {
+            std::cerr << "Plugin conflict: Cannot load both account_history plugin and elasticsearch plugin\n";
+            return 1;
+         }
+
+         std::for_each(plugins.begin(), plugins.end(), [node](const std::string& plug) mutable {
+             if (!plug.empty()) {
+                node->enable_plugin(plug);
+             }
+         });
+      }
+      else {
```

Comment:
Perhaps remove this `else` block and set a default value in line 70 instead?

Reply:
nice one, added at https://github.com/bitshares/bitshares-core/pull/1437/commits/ce35a798669660fa1515f536a9e9b319eae47d48